README.md

```diff
@@ -145,6 +145,20 @@ elements via selectors:
 ```
 
 
+### Accessibility
+
+Simplete will add the ARIA roles and attributes necessary in order for the
+autocomplete component to work with assistive technologies.
+
+While it is possible to [customize](#customization) the component with
+arbitrary HTML, we recommend using a semantic HTML list (`ol` or `ul`) with a
+corresponding
+[aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label),
+because we have performed screenreader tests for this markup.
+The `aria-label` was necessary in order for the list of suggestions to be
+announced as a list in NVDA.
```

Comment:
```suggestion
```suggestion
because we have performed screenreader tests for this markup
(`aria-label` was necessary in order for the list of suggestions to be
announced as a list in NVDA).
```
This reads a little smoother for an external audience? At least I had to think twice about what that last sentence tells me as a user - in fact, perhaps it's TMI; too detailed, more confusing than helpful?